src/components/AddressView/index.css

```diff
@@ -0,0 +1,22 @@
+.trader-account-label {
+  display: flex;
```

Comment:
for now we are using structure like AddressView/AddressView.css

I understand some benefits of using index files, but, so far would probably better to keep code style as in the rest project